### ISSUE TITLE
NameError: global name 'FileNotFoundError' is not defined

### ISSUE THREAD

**Issue by torbengb** [torbengb/bank2ynab#160]
Script language: Python 2.7
Operating system: Linux
OS Version: Ubuntu 16.04 LTS

What did you DO? (steps to reproduce)

I have just grabbed the latest ZIP file of the develop branch, then tried running the script with Python2.

torben@torben-desktop$ python2 bank2ynab.py
Traceback (most recent call last):
  File "bank2ynab.py", line 461, in <module>
    b2y.run()
  File "bank2ynab.py", line 443, in run
    files = bank.get_files()
  File "bank2ynab.py", line 310, in get_files
    except FileNotFoundError:
NameError: global name 'FileNotFoundError' is not defined
torben@torben-desktop$ 

What did you EXPECT to happen?
Bank files converted to YNAB CSV.

What ACTUALLY happened?
Error:
  File "bank2ynab.py", line 310, in get_files
    except FileNotFoundError:
NameError: global name 'FileNotFoundError' is not defined

This seems to be a not-new problem? And it's specific to Python 2.x (which I was deliberately testing) because when I ran the same with python3 it worked as expected. Forgive me for assigning it to you directly, nocalla :-)

**Comment by nocalla** [comment]
It's interesting that Travis didn't pick up on this! Is it only when you
don't have a target file in the folder it's searching?

**Comment by torbengb** [comment]
Yes that's odd -- I did have a freshly downloaded source CSV file. It got correctly processed with the py3 attempt I made after the failed py2 attempt.

**Comment by nocalla** [comment]
I've made Travis just run bank2ynab.py himself, as well as the test suite, which has triggered this error in 2.7. As well as unearthing further problems, but I think I can fix those. :-)

**Comment by nocalla** [comment]
Fix #160 by the following in the Python 2 checking at the top of the script. Travis works fine now. As an added benefit, he actually runs the full script, so random syntax errors we don't think of testing will get flagged!

```
try:
    FileNotFoundError
except NameError:
    FileNotFoundError = OSError
```
